gui/launcher.cpp

```diff
@@ -238,29 +238,21 @@ void LauncherDialog::build() {
 #endif
 	if (!g_system->hasFeature(OSystem::kFeatureNoQuit))
 		new ButtonWidget(this, _title + ".QuitButton", _("~Q~uit"), _("Quit ScummVM"), kQuitCmd);
+
 	new ButtonWidget(this, _title + ".AboutButton", _("A~b~out"), _("About ScummVM"), kAboutCmd);
-	if (g_system->getOverlayWidth() > 320)
-		new ButtonWidget(this, _title + ".OptionsButton", _("Global ~O~ptions..."), _("Change global ScummVM options"), kOptionsCmd);
-	else
-		new ButtonWidget(this, _title + ".OptionsButton", _c("Global ~O~pts...", "lowres"), _("Change global ScummVM options"), kOptionsCmd);
+	new ButtonWidget(this, _title + ".OptionsButton", _("Global ~O~ptions..."), _("Change global ScummVM options"), kOptionsCmd, 0, _c("Global Opts...", "lowres"));
```

Comment:
I know we had a discussion about ellipsis on buttons before, but now I'm wondering if we should have them in the lowres labels too

Reply:
Yes, it makes sense to have the same behavior for lowres as we do for normal resolutions

Reply:
Why "Global ~O~pts..." became "Global Opts..."?